package.json

```diff
@@ -65,7 +65,7 @@
 
         "lint": "npm run check-version && npm run eslint && npm run jscs && npm run complexity",
         "check-version": "node tools/check-version.js",
-        "jscs": "jscs esprima.js test/*test.js",
+        "jscs": "jscs esprima.js && find test/fixtures -name *.result.js | xargs jscs",
```

Comment:
I think you can use glob pattern here, e.g. `jscs  test/fixtures/**/*.result.js`. @mikesherov Can you confirm?